docs/synapse/userguides/storm_ref_data_mod.rstorm

```diff
@@ -506,34 +640,47 @@ Add Tag Timestamps or Tag Properties
 
 Synapse supports the use of :ref:`tag-timestamps` and :ref:`tag-properties` to provide additional context to tags where appropriate.
 
+.. TIP::
+  
+  You can optionally use the :ref:`edit-try` ( ``+?#`` ) when setting or modifying tag timestamps or tag properties.
+  Note that the tag and tag timestamp(s) or the tag and tag property are evaluated as a whole; if any
+  part of the tag expression is invalid, the full edit operation will fail. For example, when attempting
+  to add a tag with timestamps where the tag is valid but the timestamp values are not, neither the tag nor the
+  timestamps will be applied.
+
 **Syntax:**
 
 Add tag timestamps:
 
-*<query>* **[ +#** *<tag>* **=** *<time>* | **(** *<min_time>* **,** *<max_time>* **)** ... **]**
+*<query>* **[ +#** | **+?#** *<tag>* **=** *<time>* | **(** *<min_time>* **,** *<max_time>* **)** ... **]**
 
 Add tag property:
 
-*<query>* **[ +#** *<tag>* **:** *<tagprop>* **=** *<pval>* ... **]**
+*<query>* **[ +#** | **+?#** *<tag>* **:** *<tagprop>* **=** *<pval>* ... **]**
```

Comment:
Tagprops actually use the `?=` try operator like props
```suggestion
*<query>* **[ +#** *<tag>* **:** *<tagprop>* **=** | **?=** *<pval>* ... **]**
```

Reply:
:+1: